kansha/card/comp.py

```diff
@@ -360,6 +290,7 @@ def __init__(self, target, *args):
         """
         super(CardWeightEditor, self).__init__(target, self.fields)
         self.weight.validate(self.validate_weight)
+        self.action_button = component.Component(self, 'action_button')
```

Comment:
Well, this is required for the comp.call inside that view to work. It happens we have to persist the component with its view model othewise the comp.call won't change the view.